keadm/cmd/keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -78,6 +78,10 @@ func (cu *KubeCloudInstTool) RunCloudCore() error {
 		return fmt.Errorf("not able to create %s folder path", KubeEdgeLogPath)
 	}
 
+	if err := os.MkdirAll(KubeEdgeUsrBinPath, os.ModePerm); err != nil {
```

Comment:
Only for cloudcore?

Reply:
And does the new dir in system `PATH` env?

Reply:
KubeEdgeUsrBinPath is `/usr/local/bin` which exist in `PATH` as default.

Edgecore stores in `/etc/kubeedge`, do we unify them?

Reply:
> KubeEdgeUsrBinPath is /usr/local/bin which exist in PATH as default.

The path is not existed in the host, how can it  exist in PATH..?

>Edgecore stores in /etc/kubeedge, do we unify them?

Sure, the package shouldn't place under /etc

Reply:
Put the binary under `/usr/local/bin`, and use absolute path to call.